### ISSUE TITLE
Unable to open report in WCAG EM Report Tool

### ISSUE THREAD

**Issue by jkodu** [w3c/wcag-em-report-tool#324]
Some manual assessments of a given [ACT Rule](https://act-rules.github.io/pages/about) were conducted using the WCAG EM Report Tool. Such assessments were later submitted as implementations to the ACT Rules community.

In order to conduct an assessment, we provide a WCAG EM Report compliant JSON data for each of the rules. For example - https://act-rules.github.io/rules/97a4e1#useful-links

![image](https://user-images.githubusercontent.com/20978252/73347678-ea746d80-427f-11ea-8665-5f303070e6b0.png)

An example can be found here - https://act-rules.github.io/testcases/97a4e1/rule-97a4e1-testcases-for-em-report-tool.json

These used to work fine, and several implementation reports were generated in the past.

It looks like since the 2.0.0 release we are running into an error, when trying to open a report. The error in question looks like below.

![image](https://user-images.githubusercontent.com/20978252/73347785-17c11b80-4280-11ea-979c-30f19692d7e9.png)

I tried to debug, but there are no logs or stack trace. 
Any help appreciated.

**Comment by rvantonisse** [comment]
Looking into this, thank you for reporting.

**Comment by rvantonisse** [comment]
Hey @jkodu ,

> Some manual assessments of a given ACT Rule were conducted using the WCAG EM Report Tool.

The file you provided as example, https://act-rules.github.io/testcases/97a4e1/rule-97a4e1-testcases-for-em-report-tool.json, contains WCAG 2.1 asserted Assertions with `test: WCAG2:WCAG2:<WCAG21_ID>` where `<WCAG21_ID>` is replaced with the actual id.
Also, what is causing the bug, the data has assertions put under the property `auditResults`. The tool itself exports assertions under `auditResult`.
This data was not originally generated by the previous version of the reporter, but manually added after exporting from the previous version.

I would disencourage any modification of data if you want to be certain it will be opened by the reporter later.

I will close this issue since this is not a bug caused by the reporter but caused by manually modifying the reporter generated json.
